doc/harmony-ap_logging_user_guide.md

```diff
@@ -0,0 +1,371 @@
+2. Save the changes and restart the Harmony AP service:
+   ```bash
+   systemctl restart harmony-ap
+   ```
+
+#### 3.1.3 Log rotation
+
+Log rotation is the process of managing log files to prevent them from becoming too large and consuming too much disk space. Log rotation is essential to ensure that log files are manageable and that the system does not run out of disk space.
+
+By default, AP linux package uses the same configuration for all the logs saved into files:
+
+- Maximum file size: Each log file can grow up to 30MB before a new file is created.
+- Maximum history: Up to 60 days worth of log files are retained.
+- Total size cap: The total size of all log files is capped at 20GB. If this limit is reached, older files are deleted to make room for new ones, independently of the maximum history setting.
+
+The following example shows the default configuration for log rotation:
+
+```xml
+<rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+  <fileNamePattern>${logdir}/statistics-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+  <maxFileSize>30MB</maxFileSize>
+  <maxHistory>60</maxHistory>
+  <totalSizeCap>20GB</totalSizeCap>
+</rollingPolicy>
+```
+
+### 3.2 Logback configuration in Harmony AP Docker image
+
+The default configuration file for Logback is available in the [logback.xml](https://github.com/nordic-institute/harmony-common/blob/main/packaging/ap/docker/logback.xml).
+
+#### 3.2.1 Viewing logs
+
+Logs are sent to the standard output (`stdout`), this allows the logs to be captured by Docker and viewed using the `docker logs` command:
+
+```bash
+docker logs -f <container-name>
+```
+
+Since Docker handles log files, Logback does not save logs to the file system.
+
+#### 3.2.2 Modifying the Logback configuration
+
+Logback configuration in the Harmony AP Linux package is located at `/var/opt/harmony-ap/etc/logback.xml`. To modify the Logback configuration in the Harmony AP Docker image, create a custom image with the modified configuration:
+
+```Dockerfile
+FROM niis/harmony-ap:2.3.0
+
+COPY logback.xml /opt/harmony-ap/setup/logback.xml
+```
+
+When the container starts, the file is moved from the `setup` directory to `/var/opt/harmony-ap/etc/logback.xml`.
+
+## 4 Enable logging of full messages for debugging
+
+By default, AP logs do not include messages sent/received by the Access Point. In some situations, logging the full messages might be required for debugging purposes. In that case, the logging configuration can be changed.
+
+The following steps describe how to enable logging of full messages in Harmony AP:
+
+1. Open the Logback configuration file (`logback.xml`). Refer to the section on modifying the Logback configuration for the [Linux package](#312-modifying-the-logback-configuration) or [the Docker image](#322-modifying-the-logback-configuration) to locate the file.
+2. Locate the logger configuration for the `org.apache.cxf` package.
+3. Change the log level from `WARN` to `INFO`:
```

Comment:
After changing the file restarting the system service / container is probably required to apply the change?